ml-agents/mlagents/trainers/directory_utils.py

```diff
@@ -0,0 +1,42 @@
+import os
+from mlagents.trainers.exception import UnityTrainerException
+
+
+def handle_existing_directories(
```

Comment:
`validate_existing_directories`? "handle" is pretty vague, especially since this is on its own now.